build/bin/sage-spkg

```diff
@@ -646,40 +618,24 @@ unset V
 
 # First uninstall the previous version of this package, if any
 if [ "$KEEP_EXISTING" != "yes" ]; then
-    sage-spkg-uninstall "$PKG_BASE" "$SAGE_INST_LOCAL" --log-directory .
+    sage-spkg-uninstall "$PKG_BASE" "$SAGE_INST_LOCAL"
 fi
 
 # To work around #26996: Create lib and set a symlink so that writes into lib64/ end up in lib/
 (mkdir -p "$SAGE_DESTDIR_LOCAL/lib" && cd "$SAGE_DESTDIR_LOCAL" && ln -sf lib lib64)
 
 # Run the pre-install script, if any
 if [ -f spkg-preinst ]; then
-    sage-logger -p "$SAGE_SUDO ./spkg-preinst" spkg-preinst.log
-    if [ $? -ne 0 ]; then
-        error_msg "Error running the preinst script for $PKG_NAME."
-        exit 1
-    fi
+    sage-logger -P spkg-preinst "$SAGE_SUDO ./spkg-preinst" spkg-preinst || exit_with_error_msg "Error running the preinst script for $PKG_NAME."
 fi
```

Comment:
`spkg-preinst.log` is generated. No?

Reply:
Thanks for catching this; fixed in 7efeba390b3